src/pages/settings/settings.html

```diff
@@ -8,23 +8,29 @@
     <ion-list-header>
       {{ 'SETTINGS.PROFILE' | translate }} <span *ngIf="getUserEmail()">({{ getUserEmail() }})</span>
     </ion-list-header>
+
+    <ion-item>
+      <ion-toggle [(ngModel)]="hideName" (ionChange)="triggerHideName()"></ion-toggle>
+      <ion-label>
+        Hide name in rankings
+      </ion-label>
+    </ion-item>
+  </ion-list>
+  <ion-list no-lines>
+    <ion-list-header>
+      {{ 'SETTINGS.SUPPORT' | translate }}
+    </ion-list-header>
     <button ion-item (click)="goToTutorial()">
       <ion-icon name="medkit" item-left></ion-icon> {{ 'SETTINGS.TUTORIAL' | translate }}
     </button>
     <button ion-item (click)="goToTermConditions()">
       <ion-icon name="clipboard" item-left></ion-icon> {{ 'SETTINGS.TERMS' | translate }}
     </button>
-    <!-- <button ion-item (click)="goLeaderBoardSettings()">
```

Comment:
Hi @imkven, do we need to remove this commented code?

Reply:
This comment has removed, right?

Reply:
Hi Kven, I just found that those code seems to be removed already in actual codebase. Thats great, thx bro.